lib/rubocop/cop/salsify/rspec_doc_string.rb

```diff
@@ -28,12 +28,41 @@ class RspecDocString < RuboCop::Cop::RSpec::Base
         DOUBLE_QUOTE_MSG =
           'Example Group/Example doc strings must be double-quoted.'
 
-
-        DOCUMENTED_METHODS = RuboCop::ConfigLoader.default_configuration.for_department('RSpec')
-                               .fetch('Language')
-                               .values_at('ExampleGroups', 'Examples', 'SharedGroups', 'Includes')
-                               .flat_map { |element| element.values.flatten }
-                               .map(&:to_sym)
+        DOCUMENTED_METHODS = [
```

Comment:
This is a mess. The config data structure has a ton of linking and such between the various types. I kept having false starts at producing the same result as before, so I just copied the values. Sub-optimal, but I ran out of patience for trying to tie us to the config format of rubocop-rspec.

Reply:
I struggled to get the structure right, glad you got it working. That said, sounds like you and I agree that hardcoding this is the more stable approach, so I'll leave it as-is unless someone disagrees.